package.json

```diff
@@ -30,7 +30,7 @@
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^3.6.0",
     "eslint-plugin-prettier": "^2.7.0",
-    "jest": "^26.6.3",
+    "jest": "^27.4.5",
```

Comment:
Made this change just to keep the top-level package.json in sync with the two package-level files.